Gordon360/Controllers/ProfilesController.cs

```diff
@@ -211,34 +211,10 @@ public async Task<ActionResult<JObject>> GetMyImgAsync()
             var username = AuthUtils.GetUsername(User);
             var photoModel = await _profileService.GetPhotoPathAsync(username);
             JObject result = new JObject();
+            var defaultImgPath = _config["DEFAULT_IMAGE_PATH"] + photoModel.Img_Name;
+            result.Add("def", await GetProfileImageOrDefault(defaultImgPath));
+            return Ok(result);
```

Comment:
Why are you making major changes to the Profile Controller for 360 here? Your project branches should be focused on your project alone. We welcome updates to the codebase, but those should be merged into develop from their own branches first.

Reply:
> Why are you making major changes to the Profile Controller for 360 here? Your project branches should be focused on your project alone. We welcome updates to the codebase, but those should be merged into develop from their own branches first.

This was an accidental push, I forgot I modified this so I could check 360UI without pulling my hair out. I'll revert it

Reply:
> I believe this was done to fix a 15 second delay we face using our own 360recim.gordon.edu. Good point though that this can and probably should be done as a general codebase update (separate branch and PR into develop), but this does have direct effect on our project unfortunately.

This function is modified in our server, but not supposed to be on a branch